matrix-meetings-bot/test/MeetingService.test.ts

```diff
@@ -1804,6 +1804,57 @@ describe('test relevant functionality of MeetingService', () => {
         callInfo(0, 3, StateEventName.IO_ELEMENT_WIDGETS_LAYOUT_EVENT)
       ).toStrictEqual(expected);
     });
+
+    test('update room layout when widgets change', async () => {
+      const parentRoom: any = create_test_meeting(
+        CURRENT_USER,
+        PARENT_MEETING_ROOM_ID,
+        null,
+        [WidgetType.COCKPIT, WidgetType.BREAKOUT_SESSIONS]
```

Comment:
You should be able to do `[WidgetType.COCKPIT, WidgetType.BREAKOUT_SESSIONS, 'poll']` here. Then you add the `jitsi` widget and we would expect to have the layout for `["jitsi", "poll"]` installed as defined in the `test_default_widget_layouts.json.json`.